magister-calendar.js

```diff
@@ -409,10 +468,16 @@ function parseAppointments(appointments, currentcourse) {
       appointment.prefix = "";
     }
 
+    // Make the title a bit more pretty
+    var title = appointment.description.split("-")[0].trim();
+    if (TITLES[title] != undefined) {
```

Comment:
This will not work if the index is actually `undefined`. Please use [`typeof`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof):

```suggestion
if (typeof TITLES[title] !== 'undefined') {
```